examples/inverse-iteration/inverse-iteration.cpp

```diff
@@ -126,7 +127,7 @@ int main(int argc, char *argv[])
     }();
     auto y = clone(x);
     auto tmp = clone(x);
-    auto norm = clone(one);
+    auto norm = gko::initialize<real_vec>({1.0}, exec);
     auto inv_norm = clone(this_exec, one);
```

Comment:
I think technically this should also be of type `real_vec`, though it would not make a difference here.

Reply:
I had it it like that, but since we use `inv_norm` to scale `x`, we need it in the same type `x->scale(lend(clone(exec, inv_norm)));`:
https://github.com/ginkgo-project/ginkgo/pull/612/files/3354330466daedb07cf8ed3221d0f287f758a78b#diff-8c65e0d7c939b9652bd35aae6dcef079R147

Reply:
Ah, right. I forgot about that.